website/static/js/folderPickerNodeConfig.js

```diff
@@ -397,6 +400,15 @@ var FolderPickerViewModel = oop.defclass({
             this.cancelSelection();
         }
     },
+    destroyPicker: function() {        
+        this.folderpicker.grid.destroy();
```

Comment:
Adding a `FolderPicker#destroy` method would be preferable to calling the FolderPicker.grid's destroy method for [looser coupling](https://en.wikipedia.org/wiki/Law_of_Demeter).